src/github.com/matrix-org/rageshake/submit.go

```diff
@@ -41,10 +41,12 @@ type submitServer struct {
 }
 
 type payload struct {
-	Text      string     `json:"text"`
-	Version   string     `json:"version"`
-	UserAgent string     `json:"user_agent"`
-	Logs      []logEntry `json:"logs"`
+	Text      string            `json:"text"`
+	AppName   string            `json:"app"`
+	Version   string            `json:"version"`
+	UserAgent string            `json:"user_agent"`
+	Logs      []logEntry        `json:"logs"`
+	Data      map[string]string `json:"data"`
```

Comment:
> `data`: a set of arbitrary name/value pairs

If so, ITYM `map[string]interface{}` otherwise the server will fail to decode numbers, arrays, objects, etc.